taxi/src/test/java/com/epam/training/taxi/MainTest.java

```diff
@@ -0,0 +1,18 @@
+package com.epam.training.taxi;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class MainTest {
```

Comment:
The else block of the condition in the `Main` class is not covered by unit tests. https://github.com/epam-deik-cooperation/epam-deik-java-dev/blob/45557997829f186056daeff6987caeac1b21c758/taxi/src/main/java/com/epam/training/taxi/Main.java
Could you please cover it? 
It will be testable if the Calculator dependency (or another object which contains a Calculator object) will be injected into the `Main` class instead of creating the instance inside the `main(` method.
There is a related comment on how to solve the tight coupling problem: https://github.com/epam-deik-cooperation/epam-deik-java-dev/pull/12/files#r740777999